k8s/production/hpa.yaml

```diff
@@ -1,13 +0,0 @@
-apiVersion: autoscaling/v1
-kind: HorizontalPodAutoscaler
-metadata:
-  name: salesforce-connector
-  namespace: gfw
-spec:
-  scaleTargetRef:
-    apiVersion: apps/v1beta1
-    kind: Deployment
-    name: salesforce-connector
-  minReplicas: 2
-  maxReplicas: 6
-  targetCPUUtilizationPercentage: 50
```

Comment:
You removed the production hpa file, was this intended?

Reply:
yes. Let's start small and cheap, and if we need more pods, we'll deploy them later